src/test/resources/edu/hm/hafner/coverage/parser/metrics/metrics.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<metrics version="7.5.0-SNAPSHOT" timestamp="2024-08-12T11:37:17.394" source="PMD">
+  <package fqcn="edu.hm.hafner.util">
```

Comment:
While I understand that technically the name is a `fqcn` but for the user it might be simpler if just `name` is used. But this is not-blocking.